src/Components/Form/FormFields/Autocomplete.tsx

```diff
@@ -14,6 +14,7 @@ type AutocompleteFormFieldProps<T, V> = FormFieldBaseProps<V> & {
   optionLabel: OptionCallback<T, string>;
   optionValue?: OptionCallback<T, V>;
   optionIcon?: OptionCallback<T, React.ReactNode>;
+  multiple?: boolean;
```

Comment:
@Ashesh3 there's already an `AutocompleteMultiSelect` form field component. Separate components to ensure proper type safety and simplified logic.

Reply:
Oh, I totally missed the `AutocompleteMultiSelect`  😓